### PR TITLE
Controls bar and page header

### PR DESCRIPTION
**Pull request by g4floyd**

### CODE REVIEW

File: client/drum-pad.html [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
+			<div
 				class="play-pause-button"
 				ng-class="beatIsPlaying ? 'pause' : 'play'"
-				ng-click="startOrStopBeat()"
-				alt="Play or pause button.">
-			<label>BPM:</label>
-			<input type="number" class="tempo input-" ng-model="tempo" min="60" max="240"></input>
+				ng-click="startOrStopBeat()">
+			</div>
+			<div class="control-variable">
```
Comment (johnjoseph3) [inline client/drum-pad.html]: What does control variable mean?

File: client/stylesheet.less [hunk 2]
```diff
@@ -17,35 +19,80 @@ body {
...
 		}
 		&.pause {
 			background-image: url('../assets/images/pause-button.svg');
-			background-size: 100% 100%;
+			background-size: 70% 70%;
+			background-color: #d32f2f;
+		}
+	}
+	.control-variable {
+		// flex-grow: 1;
```
Comment (johnjoseph3) [inline client/stylesheet.less]: Can remove this comment.

File: client/stylesheet.less [hunk 3]
```diff
@@ -17,35 +19,80 @@ body {
...
-	}
+	background-color: @backgroundColorControlBar;
+	height: 50px;
+	width: 100%;
+	margin: -20px 0px 0px -20px;
+	padding-right: 40px;
+	// label {
+	// 	color: white;
+	// 	margin-right: 5px;
+	// }
```
Comment (johnjoseph3) [inline client/stylesheet.less]: Can remove comment.

File: client/drum-pad.html (hunk 1 above)
Comment (g4floyd) [reply client/drum-pad.html]: control-variable should be a container for each of the inputs that will go there. There will be one for tempo, and plans for a number of steps variable, and possibly a time signature one. 

Do you mainly have issue with the name, or with the new class?

File: client/drum-pad.html [hunk 4]
```diff
@@ -1,13 +1,15 @@
...
+				<label>BPM:</label>
+				<input type="number" class="tempo input-" ng-model="tempo" min="60" max="240"></input>
```
Comment (johnjoseph3) [inline client/drum-pad.html]: Can you remove the "input-" class? I think that might be a mistake.

File: client/drum-pad.html (hunk 1 above)
Comment (johnjoseph3) [reply client/drum-pad.html]: Could it just be called control? Does the variable part add anything to the way you would understand it as a reader?

**Comment by g4floyd** [comment]
Made the changes and pushed. 😀

File: client/drum-pad.html (hunk 1 above)
Comment (g4floyd) [reply client/drum-pad.html]: Ahh, gotcha. Yeah, thats fine with me. I'll make the change.

**Pull request merged**